src/main/java/com/hackclub/hccore/commands/LocCommand.java

```diff
@@ -32,10 +32,7 @@ public LocCommand(HCCorePlugin plugin) {
     private String getArgsAsString(String[] args, int startIndex) {
         if (startIndex > args.length - 1) return ""; // Invalid starting index
         if (args.length < 2) return ""; // Only one arg, hence nothing to concat
-        String arguments = args[startIndex];
-        for (int i = startIndex+1; i < args.length; ++i) {
-            arguments = String.join(" ", arguments, args[i]);
-        }
+        String arguments = String.join(" ", Arrays.copyOfRange(args, startIndex, args.length));
```

Comment:
Can you just move this one line to line 52?

Reply:
How would that help? It would defeat the purpose of the helper function. Also that looks like outdated code since there's a for loop instead of the Arrays.copyOfIndex thing

Reply:
I was referring to this line: https://github.com/wei2gao/HCCore/blob/14dfad7fe07bd6ffd4b90c9dad6f1155eeef5205/src/main/java/com/hackclub/hccore/commands/LocCommand.java#L35. There's no need for the helper function because the built-in functions already check it.

Reply:
For what it's worth, having the helper function gives us more flexibility in extracting arguments as a string. The case in #69 has the argument string starting at index 2 instead of 1, and it's more convenient to call the helper than to retype this line.